cmd/plume/prerelease.go

```diff
@@ -182,7 +183,8 @@ func runCLPreRelease(cmd *cobra.Command) error {
 	}
 
 	// Sanity check!
-	if vertxt := src.Object(src.Prefix() + "version.txt"); vertxt == nil {
+	vertxt := src.Object(src.Prefix() + "version.txt")
+	if strings.Contains(spec.SourceURL(), privateBucketSuffix) == false && vertxt == nil {
```

Comment:
Nit: Since it's not a suffix when used with `Contains`, I would do:
```suggestion
	if !strings.Contains(spec.SourceURL(), "private") && vertxt == nil {
```

Reply:
The variable is to be more explicit, and to explain the item we are searching for is the suffix for private buckets.